### DISCUSSION TITLE
Hourly Data Available?

### DISCUSSION THREAD

**Discussion by brooksben11** [cytech/Home-Assistant-wundergroundpws#200] [Q&A] [answered]
First off, thanks for working on this integration! I'm relatively new to it, but one of the things it seems to be lacking for my use-case is hourly forecast data. Is this a limitation of the WU API or just something this integration doesn't fetch? I'd really love to be able to switch all of my forecast data to WU only.

**Comment by cytech** [comment]
WU API does not supply hourly forecast

**Reply by abhi08638** [reply]
Just wondering if this is still true. Based on the AI overview in Google (I know to take it with a grain of salt) and the WU documentation, there is an hourly function.

The Weather Underground API does not offer a specific "hourly forecast" endpoint. However, the API provides a detailed forecast that includes hourly data. To access hourly weather information, you can request a forecast with a specific date and location, and the API will return hourly values for that day. 


https://developer.electricimp.com/libraries/webservices/wunderground

**Comment by cytech** [comment, marked as answer]
the wunderground commercial API and personal weather station API are 2 different animals..
